DotNet/proxy.ashx

```diff
@@ -377,6 +377,14 @@ public class proxy : IHttpHandler {
                 case "content-type":
                 case "transfer-encoding":
                 case "accept-ranges":   // Prevent requests for partial content
+                case "access-control-allow-origin":
+                case "access-control-allow-credentials":
+                case "access-control-expose-headers":
+                case "access-control-max-age":
+                case "access-control-allow-methods":
+                case "access-control-allow-headers":
+                case "access-control-request-method":
```

Comment:
The two `access-control-request-*` headers are "request" headers (sent from client), not "response" headers (returned from server), so maybe they can be removed from here? 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS#The_HTTP_request_headers